docs/operations/hotplug_interfaces.md

```diff
@@ -113,6 +117,40 @@ You can now check the VMI status for the presence of this new interface:
 kubectl get vmi vm-fedora -ojsonpath="{ @.status.interfaces }"
 ```
 
+### SR-IOV interfaces 
```

Comment:
I would move the SRIOV hotplug to the migration based section and also mention the SRIOV hotplug is always migration based. Even if there is multus v4.

Reply:
Done